packages/core/hooks/useMessageExternalSite.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+type MessageExternalSiteContextType = {
+    send: (url: string, message: NonNullable<unknown>) => void;
+    setOnReceive: (onReceive: ((message: unknown) => void) | undefined) => void;
+};
+
+const noop = () => undefined;
```

Comment:
consistency nit: I believe elsewhere we're using the lodash `noop`, but am also fine with leaving this as is